modules/virt-configuring-guest-memory-overcommitment.adoc

```diff
@@ -28,6 +28,7 @@ In this example, `1024M` is requested from the cluster, but the virtual machine
 told that it has `2048M` available. As long as there is enough free memory
 available on the node, the virtual machine instance will consume up to 2048M.
 +
+.Example output
```

Comment:
This isn't really output, so I'd remove `.Example output`.

Reply:
Done.